### DISCUSSION TITLE
Quoted property names in object types.

### DISCUSSION THREAD

**Discussion by JosephDenman** [tgreyuk/typedoc-plugin-markdown#619] [Help] [answered]
I have a function:

```
export function encodeCoinInfo(coin: CoinInfo): { color: Uint8Array, nonce: Uint8Array, value: bigint };
```

In `typedoc-markdown-plugin` version `4.0.0-next.53`, the documentation for this function would display:

```
encodeCoinInfo(coin): {
  color: Uint8Array;
  nonce: Uint8Array;
  value: bigint;
}
```

After upgrading to `typedoc-markdown-plugin` version `4.0.2`, the same function is displayed:

```
encodeCoinInfo(coin): {
  "color": Uint8Array;
  "nonce": Uint8Array;
  "value": bigint;
}
```

The property names are now quoted. Is there a way to configure the plugin to remove the quotes?

**Comment by tgreyuk** [comment]
Not currently but I don't think they should be quoted. Will take a look.

**Reply by tgreyuk** [reply, marked as answer]
Will go with next release. https://github.com/typedoc2md/typedoc-plugin-markdown/pull/620.
